<launch>

	<param name="angular_vel" value="2"/>

	<param name="linear_vel" value="1"/>

	<param name="k_p" value="0.8"/>

	<param name="k_d" value="0.3"/>

	<param name="max_distance" value="1"/>

	<param name="goal_ID" value="1"/>

	<node name="turtlesim" pkg="turtlesim" type="turtlesim_node"/>

    	<node name="turtle_teleop_joy" pkg="turtlesim" type="turtle_teleop_key"/>

    	<node name="turtle1" pkg="mas_framework" type="spawn_turtle.py">

        	<param name="ID" value="1"/>

    	</node>

	<node name="turtle2" pkg="mas_framework" type="spawn_turtle.py">

        	<param name="ID" value="2"/>
		<param name="x_coordinate" value="5"/>
 		<param name="y_coordinate" value="1"/>
 		<param name="theta_coordinate" value="0"/>

    	</node>

	<node name="turtle3" pkg="mas_framework" type="spawn_turtle.py">

        	<param name="ID" value="3"/>
		<param name="x_coordinate" value="1"/>
 		<param name="y_coordinate" value="1"/>
 		<param name="theta_coordinate" value="0"/>

    	</node>

	<node name="turtle4" pkg="mas_framework" type="spawn_turtle.py">

        	<param name="ID" value="4"/>
		<param name="x_coordinate" value="1"/>
 		<param name="y_coordinate" value="5"/>
 		<param name="theta_coordinate" value="0"/>

    	</node>
</launch>

